dpdbaltics.php

```diff
@@ -1086,10 +1103,15 @@ public function printLabel($idShipment)
         $shipment = new DPDShipment($idShipment);
         $format = $shipment->label_format;
         $position = $shipment->label_position;
-
+        $isAutomated = Configuration::get(Config::AUTOMATED_PARCEL_RETURN);
         try {
             /** @var ParcelPrintResponse $parcelPrintResponse */
-            $parcelPrintResponse = $labelApiService->printLabel($shipment->pl_number, $format, $position, false);
+            if ($isAutomated) {
```

Comment:
I would set this logic to new service or atleast private function so it is easer to read